README.md

```diff
@@ -8,3 +8,23 @@ This program generates a random 32 char string, and adds it to your clipboard. N
 * `go get ...`
 * `go install`
 * now, when you run `randomString` your clipboard should have a 32 character random string
+
+## Command line flag options
+###### Boolean flags can be omitted to use default value
+
+* -len **_Default: 32_**
+    - Determines the length of the string.
+    - Example: `randomString -len=20`
+* -num **_Default: true_**
+    - Determines if string should contain number characters.
+    - Example: `randomString -num=false`
```

Comment:
i don't think you implemented this flag, right?

Reply:
oh wait, it's down below in this commit.

nevermind!

Reply:
Total oversight on my part that I didn't even add numbers 😂